test-og-develop.cfg

```diff
@@ -0,0 +1,16 @@
+[buildout]
+
+extends =
+    https://raw.githubusercontent.com/4teamwork/opengever.core/master/sources.cfg
```

Comment:
This will eventually have to be modified to use `buildout-proxy` once we make the `opengever.core` repo private again.